### PR TITLE
Factorial

### PR DESCRIPTION
**Pull request by DianaVashti**
Finishes factorial(n) using recursion with tests passing.

### CODE REVIEW

File: src/factorial.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default function factorial(n){
+  if (n < 0){
+    return "Please enter a positive integer"
+  }
+
+  if (n === 1){
+    return n
+  } else
+    return n * factorial(n-1)
+}
```
Comment (devonwesley) [inline src/factorial.js:10]: Sweet this looks really nice, neat, clean and readable.

File: test/factorial_test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  })
+
+  it('should take an imput and return the numeric factorial of that input', function(){
+    expect(factorial(5)).to.deep.equal(120)
+  })
+
+  it('should return error if user puts in a negative number', function(){
+    expect(factorial(-3)).to.equal("Please enter a positive integer")
+  })
+})
```
Comment (devonwesley) [inline test/factorial_test.js:17]: Maybe like one or two more test case. but this problem's simple enough to let it go. Goodjob.

**Comment by devonwesley** [comment]
Nice. Go ahead and merge.

File: test/factorial_test.js (hunk 2 above)
Comment (DianaVashti) [reply test/factorial_test.js:17]: Thanks for the review!

**Pull request merged**